Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "pg_query"
 description = "PostgreSQL parser that uses the actual PostgreSQL server source to parse SQL queries and return the internal PostgreSQL parse tree."
-version = "5.1.0"
+version = "5.1.0-1"
```

Comment:
The version bump would allow to release this crate without necessarily releasing a new version of `pg_query` the C library (I believe `pg_query.rs` was synced with the C library version at some point recently). The versioning is technically semver, but I'm not sure it's strictly how Rust crates do it; typically, only major, minor and patch numbers are used.